airflow/ui/src/utils/pluralize.test.ts

```diff
@@ -22,52 +22,52 @@ import { describe, expect, it } from "vitest";
 import { pluralize } from "./pluralize";
 
 type PluralizeTestCase = {
-  in: [string, number, (string | null)?, boolean?];
+  in: [string, number, string?, boolean?];
   out: string;
 };
 
-const pluralizeTestCases: PluralizeTestCase[] = [
+const pluralizeTestCases = [
   { in: ["DAG", 0, undefined, undefined], out: "0 DAGs" },
   { in: ["DAG", 1, undefined, undefined], out: "1 DAG" },
-  { in: ["DAG", 12000, undefined, undefined], out: "12,000 DAGs" },
-  { in: ["DAG", 12000000, undefined, undefined], out: "12,000,000 DAGs" },
+  { in: ["DAG", 12_000, undefined, undefined], out: "12,000 DAGs" },
```

Comment:
Big numbers like `12000` are automatically turned into `12_000` with the underscore syntax to make them more readable.